### ISSUE TITLE
viewport didnt reset after respawn

### ISSUE THREAD

**Issue by albertz** [openlierox/openlierox#177]
Converted from [SourceForge issue 2616581](http://sourceforge.net/support/tracker.php?aid=2616581), submitted by albertzeyer

The viewport changed while I was waiting for respawn, then changed to splitscreen (could be that I accidently hit spacebar or something) and when I respawned, it didn't changed back.

In addition to this, there were some graphic glitches. (See screenshot.)

**Comment by albertz** [comment]
Submitted by dark_charlie

It is correct that t didn't change back. If you open a second viewport, it should stay like that until you hide it again. Perhaps there should be a check if you opened the second viewport using the viewport keys or using viewport manager.
The graphics glitch is a bug though.

**Comment by albertz** [comment]
Submitted by albertzeyer

Not sure on that. I guess most people would consider that as a bug, when they got into another viewport suddenly in the middle of a game.

And not that the second viewport did not disappeared, it also did not switched back to my own worm (none of the two viewports) after I respawned.

I think I already commented earlier that I don't like the idea that much that you got into spectate mode while you are waiting for respawn. I think that is causing all these problemes here.

Perhaps if we just disable the spectate viewport switch keys while waiting for respawn, that would solve that issue.
